test/jsonapi/view_test.exs

```diff
@@ -290,4 +302,16 @@ defmodule JSONAPI.ViewTest do
 
     assert %{body: "Chunky"} == PostView.attributes(data, conn)
   end
+
+  test "attributes/2 can return dynamic fields" do
+    data = %{static_field: "static_field!"}
+    conn = %Plug.Conn{assigns: %{jsonapi_query: %JSONAPI.Config{}}}
+
+    assert %{
+             dynamic_1: "dynamic_1!",
+             dynamic_2: "dynamic_2!",
+             static_field: "static_field!",
+             static_fun: "static_fun!"
```

Comment:
Let's make this test differentiate values coming from `get_field/3` vs. `static_fun/2` so we are testing that `static_fun/2` is indeed called. The same goes for the `static_field` property. If the stored value of `static_field` and the generated value of `static_fun/2` are changed slightly then it will go to prove that `get_field/3` is being called for every field _except_ for `:static_fun` which is what I expect based on the code.

Reply:
Yep, good call. Updated.